modular_skyrat/modules/modular_items/lewd_items/code/lewd_clothing/lewd_shoes.dm

```diff
@@ -114,7 +114,6 @@
 	worn_icon_digi = 'modular_skyrat/modules/modular_items/lewd_items/icons/mob/lewd_clothing/lewd_shoes_digi.dmi'
 	equip_delay_other = 60
 	strip_delay = 60
-	slowdown = 1
```

Comment:
?

Reply:
I'll agree here, ?

Reply:
I removed the slowdown here, to make the shoes be on-par with the normal set of high heels should players want to just use 'em normally.

Reply:
Then you should probably mention it in your changelog as well.

But I still feel like high-heels having a slowdown kind of makes sense, you can't exactly run as fast with high-heels if you don't want to sprain your ankles.

Reply:
I do see that logic there, [as long as we ignore the ERP ration in the server and what the philosophers at Boondocks suppose](https://www.youtube.com/watch?v=PhU3IhSzarE). Though would you not say that if that's the case normal high heels, too, should carry a slowdown? Or anything other than conventional combat or athletic footwear?